src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,24 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Bulldozer;
+import core.basesyntax.model.Excavator;
+import core.basesyntax.model.Machine;
+import core.basesyntax.model.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
 
```

Comment:
let's remove this empty line

Reply:
Okay. But I thought it was worth doing a logical separation. In the first block, the output of the start, in the second block, the end of works.